### PR TITLE
Small improvements quizzes

### PR DESCRIPTION
**Pull request by sebina**
### Checklist
- [x] I run `yarn run webpack:build:main`: the project builds without errors.
- [x] I run `yarn lint`: the project builds without code style warnings.
- [x] I updated the documentation and models.
- [x] I tested the changes and all related features on the test server https://artemistest.ase.in.tum.de.
- [x] ~I added (end-to-end) test cases for the new functionality.~

### Motivation and Context
Small improvements for the quiz components 

### Description
1. Change of the default text for multiple choice and drag and drop question 
2. Markdown editor background adjustments to make it more appealing in modeling exercises
3. Loading banner for drag and drop questions 

### 
Screenshot 2019-03-30 at 20 06 15
Screenshot 2019-04-02 at 18 18 05
Screenshot 2019-04-02 at 18 19 49
Screenshot 2019-03-30 at 20 08 19
Screenshot 2019-03-30 at 20 08 44
Screenshot 2019-04-01 at 19 02 19

### CODE REVIEW

File: src/main/webapp/app/entities/modeling-exercise/modeling-exercise-dialog.component.html [hunk 1]
```diff
@@ -60,7 +60,7 @@ <h4 class="modal-title" id="myModelingExerciseLabel" jhiTranslate="arTeMiSApp.mo
         </div>
         <div class="form-group-narrow" name="problemStatement" id="field_problemStatement">
             <label class="form-control-label" jhiTranslate="arTeMiSApp.exercise.problemStatement" for="field_problemStatement">Problem Statement</label>
-            <div class="question-content">
+            <div class="question-content" style="background: #f8f9fa">
```
[the remark this answers is not in the document]
Comment (sebina) [reply src/main/webapp/app/entities/modeling-exercise/modeling-exercise-dialog.component.html]: done

File: src/main/webapp/app/entities/modeling-exercise/modeling-exercise-detail.component.html [hunk 2]
```diff
@@ -31,7 +31,7 @@ <h2><span jhiTranslate="arTeMiSApp.modelingExercise.detail.title">Modeling Exerc
             </div>
         </dd>
         <dt><span jhiTranslate="arTeMiSApp.exercise.problemStatement">Problem Statement</span></dt>
-        <dd>
+        <dd style="background: #f8f9fa; max-width: max-content">
```
[the remark this answers is not in the document]
Comment (sebina) [reply src/main/webapp/app/entities/modeling-exercise/modeling-exercise-detail.component.html]: also added to css

File: src/main/webapp/app/markdown-editor/markdown-editor.component.html [hunk 3]
```diff
@@ -36,6 +36,7 @@
     </ngb-tab>
     <!--End Editor-->
 
+
```
[the remark this answers is not in the document]
Comment (sebina) [reply src/main/webapp/app/markdown-editor/markdown-editor.component.html]: deleted

File: src/main/webapp/app/modeling-editor/modeling-editor.component.html [hunk 4]
```diff
@@ -32,10 +32,10 @@ <h2>
 
     <div class="row" *ngIf="modelingExercise">
         <div class="col">
-            <p class="mb-3">
+            <div class="mb-3">
                 <span><strong>Problem Statement:</strong></span>
-                <span [innerHTML]="problemStatement" ></span>
-            </p>
+                <div style="background: #f8f9fa"  [innerHTML]="problemStatement" ></div>
```
[the remark this answers is not in the document]
Comment (sebina) [reply src/main/webapp/app/modeling-editor/modeling-editor.component.html]: added to css

File: src/main/webapp/app/quiz/participate/drag-and-drop-question/drag-and-drop-question.component.html [hunk 5]
```diff
@@ -1,4 +1,10 @@
-<div class="dnd-question" [ngClass]="{disabled: clickDisabled && !showResult, result: showResult && !forceSampleSolution, incorrect: (score || 0) < question.score && !forceSampleSolution}">
+<div class="dnd-question"  [ngClass]="{disabled: clickDisabled && !showResult, result: showResult && !forceSampleSolution, incorrect: (score || 0) < question.score && !forceSampleSolution}">
+    <div [hidden]="loading">
+        <h2>
+            <span style="color:grey"><fa-icon icon="spinner"></fa-icon>&nbsp;<span>Please wait until the picture is loaded.</span></span>
```
[the remark this answers is not in the document]
Comment (sebina) [reply src/main/webapp/app/quiz/participate/drag-and-drop-question/drag-and-drop-question.component.html]: added to css

File: src/main/webapp/app/components/util/secured-image.component.ts [hunk 6]
```diff
@@ -35,9 +42,29 @@ export class SecuredImageComponent implements OnChanges {
...
+    // triggers the reload of the picture when the user clicks on a button
+    retryLoadImage() {
+        this.retryCounter = 0;
+        this.endLoadingProcess.emit('loading');
+        this.ngOnChanges();
+    }
+
     private loadImage(url: string): Observable<any> {
-        return this.httpClient
+        const element = this.httpClient
```
Comment (jpbernius) [inline src/main/webapp/app/components/util/secured-image.component.ts]: Why?

File: src/main/webapp/app/components/util/secured-image.component.ts [hunk 7]
```diff
@@ -22,6 +24,11 @@ export class SecuredImageComponent implements OnChanges {
     @Input()
     private src: string;
     private src$ = new BehaviorSubject(this.src);
+    private value = 0;
+    private retryCounter = 0;
+
+    @Output()
+    endLoadingProcess = new EventEmitter();
```
Comment (jpbernius) [inline src/main/webapp/app/components/util/secured-image.component.ts]: Please Type this EventEmitter. Maybe add a Enum for the Possible Return Values?

File: src/main/webapp/app/quiz/participate/drag-and-drop-question/drag-and-drop-question.component.ts [hunk 8]
```diff
@@ -24,9 +25,10 @@ window.addEventListener('touchmove', function() {});
 @Component({
     selector: 'jhi-drag-and-drop-question',
     templateUrl: './drag-and-drop-question.component.html',
-    providers: [ArtemisMarkdown, DragAndDropQuestionUtil]
+    providers: [ArtemisMarkdown, DragAndDropQuestionUtil],
 })
 export class DragAndDropQuestionComponent implements OnChanges {
+    @ViewChild(SecuredImageComponent) secureImageComponent: SecuredImageComponent;
```
Comment (jpbernius) [inline src/main/webapp/app/quiz/participate/drag-and-drop-question/drag-and-drop-question.component.ts]: Why do you need this `@ViewChild`?

Comment (maximilianmeier) [reply src/main/webapp/app/quiz/participate/drag-and-drop-question/drag-and-drop-question.component.ts]: the viewChild is used in the template to reload the image in button click

File: src/main/webapp/app/components/util/secured-image.component.ts (hunk 6 above)
Comment (sebina) [reply src/main/webapp/app/components/util/secured-image.component.ts]: we can also make a return without using an additional attribute

File: src/main/webapp/app/components/util/secured-image.component.ts (hunk 7 above)
Comment (sebina) [reply src/main/webapp/app/components/util/secured-image.component.ts]: ok - created an enum

File: src/main/webapp/app/quiz/participate/drag-and-drop-question/drag-and-drop-question.component.ts (hunk 8 above)
Comment (sebina) [reply src/main/webapp/app/quiz/participate/drag-and-drop-question/drag-and-drop-question.component.ts]: added a comment as an explanation for it

File: src/main/webapp/app/components/util/secured-image.component.ts [hunk 9]
```diff
@@ -22,6 +24,17 @@ export class SecuredImageComponent implements OnChanges {
...
+
+    // Status that is emitted to the client to describe the loading status of the picture
+    QuizEmitStatus = {
+        SUCCESS: 'success',
+        ERROR: 'error',
+        LOADING: 'loading',
+    };
+
+    @Output()
+    endLoadingProcess = new EventEmitter<string>();
```
Comment (jpbernius) [inline src/main/webapp/app/components/util/secured-image.component.ts]:
```suggestion
    endLoadingProcess = new EventEmitter< QuizEmitStatus >();
```

**Pull request merged**